### PR TITLE
Closes #1349 Google Analytics Closes #1356 Removed developer vigs and Closes 1231

### LINKED ISSUE #1349: General Issue: Remove Google Analytics
### Background Information

I accidently merged in the GA stuff before discussion was completed.  We decided that we should not use this feature for privacy issues.  

### Definition of Done

GA removed from pkg yml file.

### LINKED ISSUE #1356: Documentation:  Navbar update, Removal of Dev Vignettes
### Please select a category the issue is focused on?

Other

### Let us know where something needs a refresh or put your idea here!

* Overhaul of Navbar to have more user friendly language
* Removal of Development Guides from admiral-core

### PR DESCRIPTION
**Pull request by bms63**
…#2131 most developer functions and tests have been removed

Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/) 
- [x] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://pharmaverse.github.io/admiral/articles/unit_test_guidance.html#writing-unit-tests-in-admiral-)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiral/articles/programming_strategy.html#deprecation-1)?
- [x] Update to all relevant roxygen headers and examples 
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by bms63** [comment]
## Reminder

- [ ] Make sure to grab latest vignette updates for dev guides for deprecation strategy

**Comment by bms63** [comment]
@thomas-neitmann  I think the man issue is failing as we are not using `@devel` or the workflow is looking for `admiraldev` in the lock file?  Not totally sure?

File: NEWS.md [hunk 1]
```diff
@@ -11,6 +11,9 @@
 - All admiral utility functions are exported now such that they can be used in
 admiral extension packages (#1079)
 
+- Moved all developer-facing functions to `{admiraldev}`. This supersedes above (#1079)
+ update (#1231)
```
Comment (thomas-neitmann) [inline NEWS.md:15]: We should remove #1079 from the NEWS file and change the wording here accordingly.

**Comment by bms63** [comment]
@cicdguy @thomas-neitmann  Is this an `admiralci` issue?

image

**Comment by cicdguy** [comment]
> @cicdguy @thomas-neitmann Is this an `admiralci` issue?
> 
> image

Since you're adding

```
Imports:
    admiraldev
```

You also need to update `renv.lock`, otherwise it will not work.

**Comment by bms63** [comment]
@cicdguy @thomas-neitmann I added admiraldev to lock file, but I can't seem to rehash it.  
image

**Comment by cicdguy** [comment]
> @cicdguy @thomas-neitmann I added admiraldev to lock file, but I can't seem to rehash it. image

Yeah don't worry about the hash. It's not mandatory. It's just intended for caching.

**Comment by bms63** [comment]
@cicdguy I've noticed that the Check Templates Action has disappeared?

**Comment by cicdguy** [comment]
> @cicdguy I've noticed that the Check Templates Action has disappeared?

It's actually now run only when the PR gets approved per @thomas-neitmann's request.

**Comment by cicdguy** [comment]
> Right. But I don't see it in the list of checks anymore. In the past (like 45 minutes ago) it was greyed out until someone approved the PR.

Yep, that's how the GitHub checks API works. The greyed out behavior is expected behavior.

**Comment by bms63** [comment]
I'm referring to this one

image

**Comment by cicdguy** [comment]
Yeah let's see what happens when someone re-approves this. Let me try approving it.

**Review by cicdguy — APPROVED** [review]

**Comment by cicdguy** [comment]
There ya go.

**Comment by bms63** [comment]
Thanks!  @cicdguy You are amazing.

**Comment by bms63** [comment]
> Ah, so check template failed because the branch name ought to be `1356_1349_1231_web_and_dev@devel`

ACtually we have an issue with `negate_vars` being in admiraldev but being used in the adpp script

**Comment by bms63** [comment]
@cicdguy Could you re-approve so templates are checked again?  I pushed up some updates to the templates

**Comment by bms63** [comment]
Also, does this exit_code mean only for errors and warnings?  Or is it for errors, warnings and messages?

image

**Comment by cicdguy** [comment]
I believe it's for _anything_ that generates a non-zero exit code, since the templates are execute as vanilla R scripts.

**Comment by bms63** [comment]
@millerg23 Not sure if this is the culprit, but I have a suspicion that message from `derive_param_merge_lookup` might be causing an issue for check_template aciton.

**Comment by thomas-neitmann** [comment]
> Also, does this exit_code mean only for errors and warnings? Or is it for errors, warnings and messages?

It's only for errors. Warnings have a different exit code.

**Review by thomas-neitmann — APPROVED** [review]

**Pull request merged**